### ISSUE TITLE
Questions on the features and transductive_resnet12.py

### ISSUE THREAD

**Issue by ZhangJunshuo** [MichalisLazarou/iLPC#3]
Hi, I found that the features for WRN-28-10 need access to be downloaded, I have submitted a request on Google Drive, could you please pass it? Thank you very much! Also, when I ran the demo (python transductive_resnet12.py --dataset miniImagenet --algorithm ilpc --alpha 0.8 --K 15 --n_shots 1 --which_dataset pkl) I found that the following two imports do not execute correctly, they may be unnecessary?
1. from data_closer.datamgr import SetDataManager, TransformLoader
2. from models import res_mixup_model, wrn_mixup_model
Thanks for your attention.

**Comment by ZhangJunshuo** [comment]
I couldn't find SetDataManager, TransformLoader and res_mixup_model in the source code. Do they come from other external documents?

**Comment by zhang1hongliang** [comment]
@ZhangJunshuo, yes, I get the same issue, do you have solved this question? @MichalisLazarou，perhaps you forgot to upload these files, can you update these files as quickly as possible?
